<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"> background: qlineargradient(
        spread: pad,
        x1: 0, y1: 0, x2: 1, y2: 1,
        stop: 0 white,
        stop: 0.33 #FFC0CB,  /* hồng nhạt */
        stop: 0.66 #ADD8E6, /* xanh nhạt */
        stop: 1 #FFFACD     /* vàng nhạt */
    );</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QMainWindow {
    background: qlineargradient(
        spread: pad,
        x1: 0, y1: 0, x2: 1, y2: 1,
        stop: 0 #FFD700, /* Vàng sáng */
        stop: 0.33 #FF69B4, /* Hồng tươi */
        stop: 0.66 #1E90FF, /* Xanh dương sáng */
        stop: 1 #32CD32  /* Xanh lá cây tươi */
    );
}
</string>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>431</width>
      <height>291</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="label_4">
       <property name="styleSheet">
        <string notr="true">QLabel {
    font-weight: bold;
    color: black;
    text-align: center;
}
</string>
       </property>
       <property name="text">
        <string>Đăng Kí Tài Khoản</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tên Đăng Nhập</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="usernameInput">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Mật Khẩu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="passwordInput"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Xác Thực Mật Khẩu</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="confirm_passwordInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="registerButton">
       <property name="styleSheet">
        <string notr="true">QPushButton {
    font-weight: bold;
    color: white;
    background-color: black;
    border: 1px solid black;
    text-align: center;
}

QPushButton:hover {
    color: black;
    background-color: white;
    border: 1px solid black;
}
</string>
       </property>
       <property name="text">
        <string>Đăng Kí</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="showUserListButton">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>490</y>
      <width>211</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
    font-weight: bold;
    color: white;
    background-color: black;
    border: 1px solid black;
    text-align: center;
}

QPushButton:hover {
    color: black;
    background-color: white;
    border: 1px solid black;
}
</string>
    </property>
    <property name="text">
     <string>Show Danh Sách Người Dùng</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
